activesupport/lib/active_support/core_ext/numeric/time.rb

```diff
@@ -5,6 +5,11 @@
 require 'active_support/core_ext/date/acts_like'
 
 class Numeric
+  # Convert number to local Time.
+  def to_time
+    Time.at(self)
```

Comment:
Is there any concern that Numeric conversions to Time are always left in local (system) time?